utils/api/properties.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
+const { NEXT_PUBLIC_PROPERTIES_ENDPOINT_API } = process.env
+
+export const getProperties = async (params) => {
+  const { data } = await axios.get(
+    `${NEXT_PUBLIC_PROPERTIES_ENDPOINT_API}/?q=${params}`
+  )
+
+  return data
+}
+
+export const getProperty = async (propertyReference) => {
+  const { data } = await axios.get(
+    `${NEXT_PUBLIC_PROPERTIES_ENDPOINT_API}/${propertyReference}`
+  )
+
+  return data
+}
```

Comment:
Is there any particular reason for deleting the `/properties` and required it as part of the environment variable? Isn't better to have only one env var which will be the `API_URL`?